staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/width.go

```diff
@@ -83,9 +89,17 @@ func (e WorkEstimatorFunc) EstimateWork(r *http.Request, flowSchemaName, priorit
 	return e(r, flowSchemaName, priorityLevelName)
 }
 
+// WorkEstimator is a wrapper around WorkEstimatorFunc that also contains a function for
+// calculating ratio of estimated and maximum number of seats associated with a request.
+type WorkEstimator interface {
+	// Estimate returns the estimated work of a given request.
+	Estimate(*http.Request, string, string) WorkEstimate
+	// RelativeMaximumSeats calculates a ratio of estimated and
+	// maximum number of seats associated with a request.
```

Comment:
Again the sense of maximum in the name refers to max over initial and final; the configured bound is another matter.

Reply:
Changed to:
```
// RelativeMaximumSeats calculates a ratio of maximum (of initial and final) estimate divided by
// the configured bound on the number of seats associated with a request.
```